encrypted-media-respec.html

```diff
@@ -887,42 +926,84 @@
                 </p>
+                <ol>
+                  <li>
+                    <p>Add all the configurations for which consent has been denied to <var>restrictions</var>.</p>
+                  </li>
+                  <li>
+                    <p>Return <code>ConsentDenied</code> and <var>restrictions</var>.</p>
+                  </li>
+                </ol>
               </li>
               <li>
                 <p>
-                  If there is persisted consent for <var>origin</var> indicating that <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> are allowed in conjunction with <var>accumulated configuration</var>, return <code>Allowed</code>.
-                </p>
-                <p class="note">
-                  A previous user consent for a prompt that did not include use of a <a def-id="distinctive-identifier"></a> (with similar properties) would not be considered as covering an <var>accumulated configuration</var> that requires use of such an identifier.
+                  If there is persisted consent for <var>origin</var> indicating <var>accumulated configuration</var> is allowed, return <code>Allowed</code>.
                 </p>
               </li>
               <li>
                 <p>
-                  If any of the recommendations of <a href="#allow-identifiers-cleared">Allow Identifiers to be Cleared</a> are not supported by the User Agent, return <code>ConsentRequired</code>.
+                  If any of the following are true:
                 </p>
+                <ul>
+                  <li>
+                    <p>
+                      the <a def-id="option-distinctiveIdentifier"></a> member of <var>accumulated configuration</var> is not
+                      <a def-id="requirement-not-allowed"></a> and any of the recommendations of
+                      <a href="#allow-identifiers-cleared">Allow Identifiers to be Cleared</a> are not supported by the
+                      User Agent.
+                    </p>
+                  </li>
+                  <li>
+                    <p>
+                      the user agent requires explicit user consent for the <var>accumulated configuration</var> for other reasons.
+                    </p>
+                    <p class="note">
+                      Another reason for requiring explicit user consent may be due to the security properties of the CDM implementation.
+                    </p>
+                  </li>
+                </ul>
+                <p>then:</p>
+                <ol>
+                  <li>
+                    <p>
+                      Request user consent to use <var>accumulated configuration</var> in the <var>origin</var> and wait for the user response.
+                    </p>
+                    <p class="note">User consent to use<var>accumulated configuration</var> is dependent at least on the value of the <a def-id="option-distinctiveIdentifier"></a> member.</p>
```

Comment:
The relevant step previously said, "Request user consent to use Distinctive Identifier(s)..." I'm guessing you were noting that this more general consent must at least consider DI.

It might be more straightforward to say something like `"The consent MUST include consent to use a DI if <var>accumulated configuration</var>'s <a def-id="option-distinctiveIdentifier"></a> member is <a def-id="requirement-required"></a>.`"